Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
-  end
-
   def caveats; <<-EOS.undent
-    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni.
+    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni/decks/.
     EOS
   end
 
+  def plist; <<-EOS.undent
+   <?xml version="1.0" encoding="UTF-8"?>
+   <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+   <plist version="1.0">
+   <dict>
+     <key>Label</key>
+       <string>#{plist_name}</string>
+     <key>Program</key>
+       <string>#{opt_bin}/ooniprobe</string>
+     <key>ProgramArguments</key>
+     <array>
+       <string>-i</string>
+       <string>#{HOMEBREW_PREFIX}/share/ooni/decks/current.deck</string>
```

Comment:
Ok I understand, though wouldn't this mean that if on the next update the the plist will be referencing a path that is that of the previously installed version of the package instead of the new one?

Since plists need to be installed manually I would prefer that when an update happens it will automatically point to the newer version of the deck.